openreview/venue_request/venue_request.py

```diff
@@ -164,14 +164,55 @@ def setup_review_stage(self):
                 'default': 'No, do not email program chairs about received reviews',
                 'order': 27
             },
+            'review_rating_field_name': {
+                'description': "Name of the rating field for the review invitation, default is \'Rating\'",
+                'value-regex': '.*',
+                'required': False,
+                'default': 'Rating',
+                'order': 28
+            },
+            'review_rating_options': {
```

Comment:
as @celestemartinez says, we don't need this. They can describe this in additional review options.